sdk/src/test/scala/com/akkaserverless/javasdk/impl/eventsourcedentity/EventSourcedEntitiesImplSpec.scala

```diff
@@ -132,28 +139,6 @@ class EventSourcedEntitiesImplSpec extends AnyWordSpec with Matchers with Before
       }
     }
 
-    "fail when snapshot handler does not exist" in {
-      service.expectLogError("Terminating entity due to unexpected failure") {
-        val entity = protocol.eventSourced.connect()
-        val notSnapshot = domainLineItem("?", "not a cart snapshot", 1)
-        val snapshotClass = notSnapshot.getClass
-        entity.send(init(ShoppingCart.Name, "cart", snapshot(42, notSnapshot)))
-        entity.expect(
-          failure(s"No snapshot handler found for snapshot $snapshotClass on ${ShoppingCart.TestCartClass}")
-        )
-        entity.expectClosed()
-      }
-    }
-
-    "fail when snapshot handler throws exception" in {
-      service.expectLogError("Terminating entity due to unexpected failure") {
-        val entity = protocol.eventSourced.connect()
-        entity.send(init(ShoppingCart.Name, "cart", snapshot(42, cartSnapshot())))
-        entity.expect(failure("Unexpected failure: Boom: no items"))
-        entity.expectClosed()
-      }
-    }
```

Comment:
No more snapshot handler (for now at least, but we might want something like it, but optional, moving forward turning an internal inmem state (some convenient type) to and from the stored representation (protobuf)?

Reply:
The generated classes will be tailored to work with the state that is declared in the proto, so I don't think we can do anything here. 

Users can always transform it to something more easier to use (with setter and getters if they prefer so), but ultimately they will get proto in and need to return proto, right?

Reply:
Yes and no, that's what we generate, but if handcoding an entity, I can actually even have a primitive as state or something `@Jsonable` and that is automagically turned into proto by the serialization (https://github.com/lightbend/akkaserverless-java-sdk/blob/main/sdk/src/main/scala/com/akkaserverless/javasdk/impl/AnySupport.scala#L392). That's pretty neat, and would be nice if we could get to some way doing that with the static/codegen design as well (but not a priority right now).